tests/framework/e2e/cluster.go

```diff
@@ -640,6 +636,12 @@ func (cfg *EtcdProcessClusterConfig) EtcdServerProcessConfig(tb testing.TB, i in
 		if flag == "experimental-snapshot-catchup-entries" && !CouldSetSnapshotCatchupEntries(execPath) {
 			continue
 		}
+		if flag == "experimental-compaction-batch-limit" {
+			continue
+		}
```

Comment:
Has this already been covered in the above case `if defaultValue := defaultValues[flag]; value == "" || value == defaultValue` at line 633?